lib/ansible/modules/cloud/docker/docker_service.py

```diff
@@ -125,11 +125,16 @@
         - Use with state I(present) to restart all containers.
       type: bool
       default: 'no'
-  debug:
+  remove_orphans:
       description:
-        - Include I(actions) in the return values.
+        - Remove containers for services not defined in the compose file.
       type: bool
-      default: 'no'
+      default: false
+      version_added: 2.6
```

Comment:
This feature existed when the module was created, so `version_added` should not be used. The sanity test will fail due to this, but once that is the only failure we can safely merge. The failure won't occur after the merge.

Reply:
Done